README.md

```diff
@@ -9,55 +9,108 @@ http://52.24.131.202:8999/
 
 **NPM:** https://www.npmjs.com/package/gusto-stratera
 
+1)
+Assigned one leader, gave that person authority and responsibility, and held that person accountable for the quality of the prototype submitted.
 
-## Dependencies
-This is a node.js application which requires you to have node installed. To install node, please visit ```https://nodejs.org/download/``` and download and run the latest version of node.
+[See email](DevelopmentPoolEvidence/team/TechnicalLeadEmail.JPG)
 
-Alternatively, it's available for installation through a number of package managers, for example:
+2)
+Assembled a multidisciplinary and collaborative team that includes at a minimum of two of the labor categories limited to the Development Pool labor categories.    
 
-*chocolatey:*
-```bash
-choco install nodejs.install
-```
+Our Team consists of a DevOps Engineer and a Back-end Developer.  Both team members are experienced in the agile scrum development methodology and in playing the role of scrum master.  The DevOps Engineer plays the role of Scrum Master and also performs some of the development tasks.  The Back-end Developer performs the development quality assurance tasks such as peer reviews.
 
-*yum:*
-```bash
-sudo yum install nodejs npm --enablerepo=epel
-```
+3)
+Used at least five modern and open-source technologies, regardless of architectural layer (frontend, backend, etc.)
 
-Other package manager examples can be found on [joyent's github page](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager "joyent's github page").
- 
+| Technology | Version |
+| ---------- | ------- |
+| centos | 6.5 |
+| node | 0.12.0 |
+| express | 4.12.3 |
+| express-http-proxy | 0.6.0 |
+| istanbul | 0.3.16 |
+| mocha | 2.1.0 |
+| mocha-teamcity-reporter | 0.0.1 |
+| should | 7.0.1 |
+| supertest | 1.0.1 |
+| git |1.9.5 | 
```

Comment:
the ones in github were easy enough to verify - we have at least these 6 that were created in the last 5 years
mocha
https://github.com/mochajs/mocha/graphs/contributors
Aug 28, 2011 – Jul 6, 2015

istanbul
https://github.com/gotwarlost/istanbul/graphs/contributors
Sep 9, 2012 – Jul 6, 2015

should
https://github.com/shouldjs/should.js/graphs/contributors
Nov 14, 2010 – Jul 6, 2015

supertest
https://github.com/visionmedia/supertest/graphs/contributors
Jun 24, 2012 – Jul 4, 2015

mocha-teamcity-reporter
https://github.com/travisjeffery/mocha-teamcity-reporter/graphs/contributors
Sep 22, 2013 – Jul 6, 2015

express-http-proxy
https://github.com/villadora/express-http-proxy/graphs/contributors
May 25, 2014 – Jul 4, 2015